app/angular/src/client/preview/types-6-0.ts

```diff
@@ -33,6 +33,7 @@ export type Parameters = DefaultParameters & {
   /** Uses legacy angular rendering engine that use dynamic component */
   angularLegacyRendering?: boolean;
   component: unknown;
+  bootstrapOptions?: unknown;
```

Comment:
As for the type: only the CompilerOptions type is exported so instead of using only one part of the union type I went with unknown to avoid confusion.
I think bootstrapModuleOptions would be a good name bc this makes it clear where the options are used (in the method call bootstrapModule). I will change it